reference/mixin-arguments.rst

```diff
@@ -27,3 +27,22 @@ The following arguments are provided by the ``colcon-mixin`` package:
   * **debug**:
 
     - ``cmake-args``: ``['-DCMAKE_BUILD_TYPE=Debug']``
+
+It is important to note that in their current implementation, mixins do not
+possess semantic knowledge about CMake arguments that would allow them to
+execute merges of **arguments that appear multiple times** correctly.  Those
+conditions arise, for example, when **mixin1** and **mixin2** contain
+overlapping parameters, or when any **mixin** specifies an argument that also
+appears on the command line.  This situation should be avoided since one of
+the parameters will be silently suppressed and the outcome not be as expected.
+Case in point:
+
+\\--mixin debug coverage-gcc
+  Will *clash* since both mixins share the ``--cmake-args`` parameter.  One of
+  them will be ignored in the final CMake call.
```

Comment:
I think this examples is incorrect. While both mixins specify `--cmake-args` the CMake arguments are not colliding with each other and the superset of both CMake arguments should be used (as intended).